internal/pkg/pluginmanager/manager.go

```diff
@@ -20,15 +27,15 @@ func DownloadPlugins(conf *configloader.Config) error {
 		pluginFileName := configloader.GetPluginFileName(&tool)
 		if _, err := os.Stat(filepath.Join(pluginsDir, pluginFileName)); errors.Is(err, os.ErrNotExist) {
 			// plugin does not exist
-			log.Printf("=== now downloading plugin: %s, version: %s ===", pluginFileName, tool.Version)
+			log.Printf("=== downloading plugin: %s ===", pluginFileName)
 			err := dc.download(pluginsDir, pluginFileName, tool.Version)
 			if err != nil {
 				return err
 			}
-			log.Printf("=== plugin: %s, version: %s downloaded ===", pluginFileName, tool.Version)
+			log.Printf("=== plugin: %s has been downloaded ===", pluginFileName)
```

Comment:
What is the reason to remove the `version` from the log? Do we need it or not?

Reply:
deleted it for the version info has contained in the filename.